android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -718,12 +718,7 @@ public static RelativeLayout.LayoutParams getKeyboardLayoutParams() {
    * @param bottom int padding to account for navigation bar on bottom of screen
    */
   public static void applyInsetsToKeyboard(int left, int right, int bottom) {
-    // System keyboard
-    if (SystemKeyboard != null) {
-      applyInsetsPaddingToKeyboardView(SystemKeyboard, left, right, bottom);
-    }
-
-    // In-App Keyboard
+    // Only applies to In-App keyboard
```

Comment:
Would it be worth adding a reference to this PR / its issue as part of this comment?

Reply:
```suggestion
    // Only applies to In-App keyboard #14619
```